RevenueCatUI/Templates/V2/ViewHelpers/BackgroundStyle.swift

```diff
@@ -47,7 +46,26 @@ fileprivate extension View {
     func apply(backgroundStyle: BackgroundStyle) -> some View {
         switch backgroundStyle {
         case .color(let color):
-            self.background(color.toDynamicColor())
+            switch color.light {
```

Comment:
My assumption is the light and dark colors must be of the same type e.g. both solid or both gradient.

Reply:
Modified to support mixed solid and gradient light/dark colors in ColorScheme